### PR TITLE
Fixed the issue "Wrong translate #3148"

### PR DESCRIPTION
**Pull request by al0cam**
Fixed the issue "Wrong translate #3148" where the string "Kopya Bilgisi" had to be renamed to "Bilgileri Kopyala"

### CODE REVIEW

File: data/translations/Internationalization_tr.ts [hunk 1]
```diff
@@ -1535,7 +1535,7 @@ Lütfen bunları yapılandırma dosyasında elle çözün.</translation>
         <location filename="../../cmake-build-debug/src/flameshot_autogen/include/ui_infowindow.h" line="124"/>
         <location filename="../../cmake-build-relwithdebinfo/src/flameshot_autogen/include/ui_infowindow.h" line="124"/>
         <source>Copy Info</source>
-        <translation>Kopya Bilgisi</translation>
+        <translation>Bilgileri Kopyalai</translation>
```
Comment (ardishco-the-great) [inline data/translations/Internationalization_tr.ts:1538]: Typo, It should be "Bilgileri Kopyala" not "Bilgileri Kopyalai"

**Review by ardishco-the-great — APPROVED** [review]
There's a typo in the string change you submitted but Bilgileri Kopyala is a more accurate translation, yeah. "Kopyala Bilgisi" is structrually and contextually wrong.

File: data/translations/Internationalization_tr.ts (hunk 1 above)
Comment (al0cam) [reply data/translations/Internationalization_tr.ts]: Damn, missed that one, thank.

Comment (ethemkaraca05) [reply data/translations/Internationalization_tr.ts]: This string also seems to be wrong. it should be changed to "Güncellemeleri otomatik denetle"

<details><summary>Screenshot</summary>

annotely_image


</details>

Comment (ardishco-the-great) [reply data/translations/Internationalization_tr.ts]: > This string also seems to be wrong. it should be changed to "Güncellemeleri otomatik denetle"
> Screenshot
> 
> annotely_image

Can confirm this

**Comment by luzpaz** [comment]
bumping

**Review by panpuchkov — APPROVED** [review]

**Review by veracioux — APPROVED** [review]

**Pull request merged**